changelog/6484.misc.md

```diff
@@ -0,0 +1,19 @@
+## Summary
```

Comment:
tbh I'm not sure what happens if you do heading in here. Might destroy the formatting when this is merged in `Changelog.mdx`. I'd vote for omitting the `##` headers

Reply:
Just h2 or the h3s too?

Reply:
I'd remove all headlines to be honest. Looking at `Changelog.mdx` all sorts of headers would destroy the list items

Reply:
Got it! Thanks for walking me through the changelog stuff. :)